README.md

```diff
@@ -0,0 +1,209 @@
+# Site Status Checker
+
+Reads web addresses from a CSV file and use curl to record their HTTP status codes.  Results are provided in a modified version of the original CSV file.  Created and tested on Fedora 32 Linux.
+
+## Usage
+
+### Installation
+
+There is no installation.  The script can be cloned or copied.  If copied make sure the new file is executable.
```

Comment:
"If copied, make..."

Reply:
Done, thanks for the corrections!  (...most of which aren't shown)